docs/scripts/config.js

```diff
@@ -11,7 +11,7 @@ const { execSync } = require('child_process')
 const fetch = require('node-fetch')
 
 const oryXVersion = execSync(
-  "cd ..; go list -f '{{.Module.Version}}' -find github.com/ory/x"
```

Comment:
Unfortunately this will be overwritten if we change the docs template. What is the underlying issue here? Is it the go version?

Reply:
Can we also update the template?

The issue is that the existing `go list` without a `-m` flag isn't module aware, when it tries to list the packages this way the following happens:
```sh
$ go list -f '{{.Module.Version}}' github.com/ory/x
package github.com/ory/x: build constraints exclude all Go files in /Users/tobbbles/code/go/pkg/mod/github.com/ory/x@v0.0.165
```

This appears to be due to github.com/ory/x not containing any Go files in the root, so `go list` doesn't deem it a valid package.

This was previously solved by just touching a go package in the ory/x root on a 'patch' branch; but IMO the issue is this incorrect listing from `go list` not resolving the _module_ version, but trying to resolve the used package version

Reply:
@zepatrik wrote this - can you chime in?

Reply:
I see https://github.com/ory/x/blob/master/package.go was added 8 months ago and that's why projects on newer ory/x packages don't receive this error, however we can't upgrade oathkeeper yet due to the viperx->koanf requirement.

fwiw, the module aware go list will also works in kratos on the newer version of ory/x

```sh
$ go list -f '{{.Version}}' -m github.com/ory/x
v0.0.272

$ go list -f '{{.Module.Version}}' github.com/ory/x
v0.0.272
```

Reply:
Ah ok nice, then we can use it with `-m`. :+1:
The right place to update this script is here: https://github.com/ory/docusaurus-template/blob/8ee6f932f6ef3edf5c471bff4eb64dcbb9d22a0c/scripts/config.js#L14